linera-storage/src/db_storage.rs

```diff
@@ -576,14 +576,14 @@ where
     }
 
     async fn write_blobs(&self, blobs: &[Blob]) -> Result<(), ViewError> {
+        if !blobs.is_empty() {
```

Comment:
```suggestion
        if blobs.is_empty() {
```

Reply:
Thank you for catching this. The fact that the CI passes indicates that the Blob may not have been tested enough.